src/m365/spo/commands/theme/theme-get.ts

```diff
@@ -62,11 +62,11 @@ class SpoThemeGetCommand extends SpoCommand {
       if (contents && contents.ErrorInfo) {
         throw contents.ErrorInfo.ErrorMessage || 'ClientSvc unknown error';
       }
-      const json2: any = await Promise.resolve(json);
+      const json2: any = await json;
       const theme = json2[6];
```

Comment:
This doesn't make much sense. Right now we are awaiting a variable that is no promise. In fact, I wonder why the code was written like that before the refactoring. I don't see a reason why the `await Promise.resolve(` was needed.
I think we can remove the lines where we create the new variable `json2` and just use the original json object.

Reply:
I wasn't able to write a test without the test using the actual environment configuration. Tried resolves, callsfake, returns... Nothing worked.. Will have to take a closer look to this one